### PR TITLE
Bump and assemble refactoring

### PR DESCRIPTION
**Pull request by acmnu**
This commit bump software to latest version. Main change is
Python 3.9. Minor part of this a lot of different small packages.

This commit adds rust and cargo as build time dependencies bescause
of new parts in crypto engine (PEP517).

This commit also change logic of assembling. Everything related to
build is in assemble dir as it was done for adss_core previously.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,24 +1 @@
...
-toml
-uwsgi
-version_utils
-yspec
-# Custom bundle libs
-apache-libcloud
-jmespath
-lxml
-pycrypto
+-r assemble/base/requirements-base.txt
```
Comment (grey-kristy) [inline requirements.txt:1]: May be yspec should be here?

Comment (acmnu) [reply requirements.txt:1]: I think it should be in separate PR

**Comment by acmnu** [comment]
bender build

**Comment by BenderArenadata** [comment]
Hey, @acmnu! Build has been launched!

**Review by grey-kristy — APPROVED** [review]

**Pull request merged**